modules/text-field/js/text-field_directive.js

```diff
@@ -93,8 +87,8 @@ angular.module('lumx.text-field', [])
                     }
 
                     $field.addClass('text-field__input');
-                    $field.on('focus', focusUpdate);
-                    $field.on('blur', blurUpdate);
+                    $field.on('focus', true, focusUpdate);
```

Comment:
@badoet, the second parameter is data in that case. http://api.jquery.com/on/#on-events-selector-data-handler